src/Model/GServer.php

```diff
@@ -560,6 +571,9 @@ private static function detect(string $url, string $network = '', bool $only_nod
 				self::detect($url, $network, $only_nodeinfo);
 			}
 			return false;
+		} elseif (Network::isUrlBlocked($url)) {
+			Logger::info('Server domain is blocked', ['url' => $url]);
+			return false;
```

Comment:
`elseif` here is superfluous since there's a `return` in the parent condition.
```suggestion
		}

		if (Network::isUrlBlocked($url)) {
			Logger::info('Server domain is blocked', ['url' => $url]);
			return false;
```

Reply:
What do you mean by "parent condition"?

Reply:
The condition where the `elseif` originated.

Reply:
I usually only make a 2nd `if()` block if there is code between the upper and the then added lower one:
````
if (someCondition) {
// Do something, like logging, closing/freeing resources
return false;
}

// Do other things:
doOtherCode();

if (otherCondition) {
// Do something, like logging, closing/freeing resources
return false;
}
````
Otherwise, when there is no code in between these two `if()` blocks, I merge them with `elseif()` or when they do the exact same thing, I merge them with `||`.

Reply:
Let me provide a more actionable suggestion.